Dockerfile

```diff
@@ -1,25 +1,43 @@
 FROM debian:buster-slim
 
 # File Author / Maintainer
-MAINTAINER AGSPhoenix
+LABEL maintainer=AGSPhoenix
+
+ARG DEBIAN_FRONTEND=noninteractive
+# Makes the apt-get process aware
+# of the docker build process env
+# so it doesn't complain as much
 
 RUN \
 # MAN folder needed for jre install
-     mkdir -p /usr/share/man/man1 \
-  && mkdir -p /sheep/cache \
-# Install JRE and curl
-  && apt-get update \
-  && apt-get install -y --no-install-recommends \
+	mkdir -p \
+		/usr/share/man/man1 \
+		/sheep/cache \
+	# Install JRE, curl and blender deps
+	&& apt-get update \
+	&& apt-get upgrade -y \
```

Comment:
Upgrading packages is generally recommended against in Dockerfiles. Because we're using Debian and not likely to get much other than security fixes, this is probably fine, but a comment to that effect couldn't hurt

Reply:
You want that to be added to the upper comment, something like `Update packages, install JRE[...]`

Reply:
Commit added